File: Client/src/BCastPacket.cpp

#include "../include/Packet.h"

BCastPacket::BCastPacket(char delAdd,string fileName):Packet(9),delAdd(delAdd),filename(fileName) {


}
char BCastPacket::getDelAdd(){
	return delAdd;
}
string BCastPacket::getString(){
	return filename;
}
BCastPacket::~BCastPacket() {

}

File: Server/src/main/java/bgu/spl171/net/api/bidi/BCastPacket.java
package bgu.spl171.net.api.bidi;

public class BCastPacket extends Packet{
	
	private String fileName;
	private byte delAdd;
	
	public BCastPacket(byte delAdd, String fileName){
		this.fileName=fileName;
		super.opCode=9;
		this.delAdd=delAdd;
	}
	
	public byte getDelAdd(){
		return delAdd;
	}
	public String getFilename(){
		return fileName;
	}
	
}
